rpcs3/Emu/RSX/Common/BufferUtils.cpp

```diff
@@ -453,6 +453,10 @@ void stream_vector(void *dst, u32 x, u32 y, u32 z, u32 w)
 	_mm_stream_si128((__m128i*)dst, vector);
 }
 
+void stream_vector(void *dst, f32 x, f32 y, f32 z, f32 w)
+{
+	stream_vector(dst, (u32&)x, (u32&)y, (u32&)z, (u32&)w);
```

Comment:
Can't it be just the same, as `u32` version but with `_mm_stream_ps` or something? Also, why not `inline` (just wondering)?